src/Microsoft.Owin.Security.Cookies.Shareable/CookieAuthenticationExtensions.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.DataProtection;
+using Microsoft.Framework.Internal;
+using Microsoft.Owin.Security.Cookies;
+using Microsoft.Owin.Security.Cookies.Shareable;
+
+namespace Owin
+{
+    public static class CookieAuthenticationExtensions
+    {
+        public static IAppBuilder UseCookieAuthentication(
```

Comment:
I'll file an issue for now, I'll do the intial end to end identity + everything sample first since that's what people actually will want to use